lib/src/store.dart

```diff
@@ -159,8 +159,8 @@ class Store<State> {
   Reducer<State> reducer;
 
   final StreamController<State> _changeController;
-  State _state;
-  List<NextDispatcher> _dispatchers;
+  late State _state;
+  late final List<NextDispatcher> _dispatchers;
```

Comment:
You *could* initialize this list here:
```dart
 final List<NextDispatcher> _dispatchers = [];
```
and change the code below to add to that list instead of creating a new list.
It would avoid the field being late, but being late is also fine.